packages/metatransaction-broadcaster/MetatransactionBroadcaster.js

```diff
@@ -0,0 +1,265 @@
+import sqlite3 from "sqlite3";
+
+const ethers = require("ethers");
+const express = require("express");
+const request = require("request-promise");
+const sqlite = require("sqlite");
+const ConsoleAdapter = require("./adapters/console").default;
+
+class MetatransactionBroadcaster {
+  /**
+   * Constructor for MetatransactionBroadcaster
+   * @param {string} privateKey              The private key of the address that executes the metatransactions
+   * @param {Object} loader                  The loader for loading the contract interfaces. Usually a TruffleLoader.
+   * @param {Object} provider                Ethers provider that allows access to an ethereum node.
+   */
+  constructor({ privateKey, loader, provider, adapter, dbPath = "./mtxCache.sqlite", port = 3000 }) {
+    this.provider = provider;
+    this.wallet = new ethers.Wallet(privateKey, this.provider);
+    this.senderAddress = this.wallet.address;
+
+    console.log("Transactions will be sent from ", this.senderAddress);
+    this.dbPath = dbPath;
+
+    this.loader = loader;
+
+    this.adapter = adapter;
+    if (typeof this.adapter === "undefined") {
+      this.adapter = ConsoleAdapter;
+    }
+
+    this.app = express();
+
+    this.app.use(function (req, res, next) {
```

Comment:
Could be neat to put this logic in a common "express utils" folder shared across packages.